README.md

```diff
@@ -107,6 +108,10 @@ For assistance with breaking changes, see [MIGRATION.md](docs/MIGRATION.md).
     # Does not fail if the artifact does not exist.
     # Optional. Default is 'false'
     overwrite:
+
+    # Whether to include hidden files in the provided path in the artifact
+    # Optional. Default is 'false'
+    include-hidden-files:
```

Comment:
Do we want to put any warning within this comment?

Reply:
Added a warning - https://github.com/actions/upload-artifact/pull/598/commits/d52396ac5d312b1bda1b9fe2ae55d862c65abd68